Dockerfile.development

```diff
@@ -0,0 +1,10 @@
+FROM ruby:2.5.0-alpine
+
+RUN apk add --no-cache build-base postgresql postgresql-dev libpq git
+
+WORKDIR /app
+COPY . /app
+RUN bundle install -j $(nproc) --quiet
+
+EXPOSE 2300
```

Comment:
It is not necessary, I think. @davydovanton

Reply:
👍